cachito/web/utils.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from flask import request, url_for
+
+
+def pagination_metadata(pagination_query):
+    """
+    Returns a dictionary containing metadata about the paginated query.
```

Comment:
Nitpick: A new line is needed after this

The first word of the summary should also be singular.

Perhaps, I should add flake8-docstrings to Cachito to enforce this.